letsdrill.rb

```diff
@@ -1,24 +1,40 @@
 def get_letter_grade(integer)
- 
-  #Put your code here!
 
+  percentageGrade = integer.to_i
```

Comment:
You can probably trust that what's coming in is an integer.  If not, it might be better to throw an exception than to blindly convert the input.